// import { errorFactory } from '@/core/shared/domain';

describe('Errors factory', () => {
  it('', () => {
    expect(true).toBe(true);
  });
  // it('Should throw the error with the given error instance', () => {
  //   const DummyError = errorFactory('Dummy Error');
  //   const t = () => {
  //     throw new DummyError('Dummy message');
  //   };
  //   expect(t).toThrow(DummyError);
  // });
  // it('Should throw the error with the given name', () => {
  //   const errorInstanceName = 'Dummy Error';
  //   const DummyError = errorFactory(errorInstanceName);
  //   try {
  //     throw new DummyError('Dummy message');
  //   } catch (error) {
  //     expect(error.name).toBe(errorInstanceName);
  //   }
  // });
  // it('Should throw the error with the given error message', () => {
  //   const errorMessage = 'Dummy message';
  //   const DummyError = errorFactory('Dummy Error');
  //   try {
  //     throw new DummyError(errorMessage);
  //   } catch (error) {
  //     expect(error.message).toBe(errorMessage);
  //   }
  // });
});
